src/Zoom/Interpolation.elm

```diff
@@ -5,25 +5,33 @@ module Zoom.Interpolation exposing (interpolate)
 
 
 type alias View =
-    { cx : Float, cy : Float, size : Float }
+    { cx : Float
+    , cy : Float
+    , size : Float
+    }
 
 
+exp : Float -> Float
 exp x =
     e ^ x
 
 
+log : Float -> Float
 log =
     logBase e
 
 
-ε2 =
+epsilon2 : Float
+epsilon2 =
     1.0e-12
 
 
-ρ =
+rho : Float
+rho =
```

Comment:
?

Reply:
The old characters resulted in parse errors for elm-analyse, see issues [here](https://github.com/stil4m/elm-analyse/issues/242) and [here](https://github.com/stil4m/elm-syntax/issues/47).